sdk/tools/azure-openrewrite/src/test/resources/migrationExamples/azure-core/sample5/v1/HttpPutExample.java

```diff
@@ -0,0 +1,19 @@
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.HttpHeaders;
+import com.azure.core.http.HttpMethod;
+import com.azure.core.http.HttpRequest;
+import com.azure.core.http.HttpResponse;
+import com.azure.core.http.netty.NettyAsyncHttpClientBuilder;
+
+public class HttpPutExample {
+    public static void main(String... args) {
+        HttpClient client = new NettyAsyncHttpClientBuilder().build();
+        HttpHeaders headers = new HttpHeaders().set("Content-Type", "application/json");
+        String jsonBody = "{\"key\":\"value\"}";
+        HttpRequest request = new HttpRequest(HttpMethod.PUT, "https://example.com", headers, jsonBody);
+
+        HttpResponse response = client.send(request).block();
```

Comment:
These samples are using the async pipeline and blocking. You'll want to make sure you also support the scenario where the sync pipeline was used.